conf.yaml

```diff
@@ -103,7 +103,7 @@ contents:
           -
             title:      Getting Started
             prefix:     en/elastic-stack-get-started
-            current:    7.6
+            current:    7.7
```

Comment:
Going forward, would it help to create a `stackcurrent` variable, like we have for `stacklive`, so we only have to change this in one place, for all books which are versioned in line with the Stack?

Reply:
Thanks @gtback. I think that's a great idea.

Reply:
Went ahead and added a `&stackcurrent` variable with 1ae5b6c. Also removed `7.6` from the `&stacklive` variable as 7.6 will no longer be maintained after 7.7 releases.